src/__tests__/helpers/hoc/WithOneChild.test.tsx

```diff
@@ -39,15 +34,15 @@ describe("withOneChild HOC", () => {
         );
         wrapper = Enzyme.mount(multipleChildrenDummy);
 
-        expect(wrapper.find("p")).to.have.lengthOf(2);
-        expect(wrapper.prop("title")).to.equal("Wizard");
-        expect(wrapper.prop("age")).to.equal("old");
+        expect(wrapper.find("p")).toHaveLength(2);
```

Comment:
Was this an autoformatter thing?  I think we should keep this using the old Chai testing syntax since there's not a real need to change it.

Reply:
we weren't using jest before, is there any reason why it should be added now?